### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,113 @@ This is particularly useful when provisioning VMs with vagrant.
 
 ## WARNING
 
-This is pretty much just a prototype at the moment. Its functionality is fairly
-limited and it's almost certainly not ready for production use.
+This is pretty much just a prototype at the moment. Its functionality is quite
+limited and it hasn't been tested on a wide variety of environments. It meets
+our fairly needs, but we wouldn't recommend using it in production without
```
Comment (JayH5) [inline README.md]: "fairly basic"?

Comment (jerith) [inline README.md]: Thanks. I edited that sentence a bunch of times and must have lost the word somewhere along the way.

File: Rakefile [hunk 2]
```diff
@@ -25,7 +25,8 @@ Rake::Task[:coverage].clear
 
 # Remove puppetlabs_spec_helper's metadata and validate tasks
 Rake::Task[:validate].clear
-Rake::Task[:metadata].clear
+# This was removed in puppetlabs_spec_helper 1.1.0
+Rake::Task[:metadata].clear if Rake::Task.task_defined?(:metadata)
```
Comment (JayH5) [inline Rakefile]: I feel like [this](https://github.com/praekeltfoundation/puppet-docker_firewall/commit/708e37fa7ff23173775e54814f0dad86adcc9696) approach is slightly more robust?

Comment (jerith) [inline Rakefile]: I'll make the change, thanks.

**Comment by JayH5** [comment]
:+1:

**Pull request merged**